Jenkinsfile

```diff
@@ -225,6 +225,7 @@ pipeline {
                                                   "src/mgmt/*.pb-c.[ch]:" +
                                                   "src/engine/*.pb-c.[ch]:" +
                                                   "src/security/*.pb-c.[ch]:" +
+                                                  "src/chk/*.pb-c.[ch]:" +
```

Comment:
Should this in fact be any `*.pb-c.[ch]` in any path or can there be files matching that pattern that should be checkpatched?

Reply:
@brianjmurrell, I tend to think it could be. Could we do that on master instead and go with this here on feature/cat_recovery? I'd prefer not to refresh the pull request for only this issue.

Reply:
Ah, yes.  I hadn't noticed this was on a feature branch.